FirebaseAuth/Sources/Backend/RPC/FIRSignUpNewUserRequest.m

```diff
@@ -38,6 +38,21 @@
  */
 static NSString *const kDisplayNameKey = @"displayName";
 
+/** @var kCaptchaResponseKey
+    @brief The key for the "captchaResponse" value in the request.
+ */
+static NSString *const kCaptchaResponseKey = @"captchaResponse";
```

Comment:
nit: can we move these definitions to FirebaseAuuth/Sources/Utilities and reuse them in the various request definitions here? 

Not sure what the convention is, feel free to skip if not straightforward to address.

Reply:
We'd like to keep RPC/ directly mapped to the backend protos, so I prefer keep it as-is.